src/Cli/UpCommand.php

```diff
@@ -39,16 +40,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             return 1;
         }
-        try {
-            $dockerComposePath = $this->getDockerComposePath($processRunner);
-            pcntl_exec($dockerComposePath, ['up']);
 
-            return 0;
+        $userInteraction->writeTitle('Starting application containers');
+
+        try {
+            $processRunner->run(new Process('docker-compose up -d'));
+            $userInteraction->writeTitle('Application containers successfully started');
         } catch (ProcessFailedException $e) {
+            echo $e->getProcess()->getOutput();
+
             return 1;
         }
+
+        // Display running containers
```

Comment:
Does it make sense to call the `ps` command here? This duplication feels like it may be problematic as changes to how the containers are output would need making in both commands.